model/clusters_mgmt/v1/event_type.model

```diff
@@ -0,0 +1,21 @@
+/*
+Copyright (c) 2021 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Representation of a trackable event.
+struct Event {
+	// Key of the event to be tracked.
+	Key String
```

Comment:
What are the valid values for the key?

Reply:
Currently we don't have a list of valid keys. I've added a regular expression check to the API merge request to ensure we can have some level of validation. Once we capture enough data about issues users are facing, we might lock in a specific set of event values.

Reply:
I'd suggest to explain that in the documentation for `Key`, including the regular expression that is used to validate it, and the maximum length.